oauth2/oauth2-core/src/test/java/org/springframework/security/oauth2/core/endpoint/ErrorResponseAttributesTest.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2012-2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.security.oauth2.core.endpoint;
+
+import org.junit.Test;
+
+/**
+ * Tests ${@link ErrorResponseAttributes}
+ *
+ * @author Luander Ribeiro
+ */
+public class ErrorResponseAttributesTest {
+
+	@Test(expected = IllegalArgumentException.class)
+	public void builderWhenCodeIsNullThenThrowIllegalArgumentException() {
```

Comment:
Please rename to **withErrorCode**WhenCodeIsNullThenThrowIllegalArgumentException